client/src/App.jsx

```diff
@@ -1,34 +1,119 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { useState, useEffect } from 'react'
+import { Flex, Container, Box, Card} from '@radix-ui/themes';
+import { Heading, Text, TextField, Button, SegmentedControl, Avatar, ScrollArea, Select, Link } from '@radix-ui/themes';
+import { AccessibilityIcon } from '@radix-ui/react-icons'
+import { makeServer } from './server'
+
+if (process.env.NODE_ENV === 'development') {
+  makeServer()
+}
 
 function App() {
-  const [count, setCount] = useState(0)
+  const [inputValue, setInputValue] = useState('');
+  const [responseData, setResponseData] = useState(null);
+
+  // Get threads for first AI msg
+  useEffect(() => {
+    handleLoad()
+  }, []);
+
+  async function handleLoad() {
+    try {
+        const response = await fetch('/api/threads', { method: "get" });
+        const data = await response.json()
+        setResponseData(data);
+    } catch (error) {
+        console.log('There was an error when getting /api/threads', error);
+    }
+  }
```

Comment:
Your API is returning a array of thread, sth like:
```
{
   threads: [{
      id: ...
    }]
}
```
```suggestion
async function handleLoad() {
      try {
        const response = await fetch('/api/threads', { method: "get" });
        const data = await response.json();
        const threads = data['threads'];
        setResponseData({...threads[0]});
    } catch (error) {
        console.log('There was an error when getting /api/threads', error);
    }
  }
```